widgets/common/src/deploymentsView/DeploymentLabeLConfigurationInput.tsx

```diff
@@ -0,0 +1,27 @@
+import type { FunctionComponent } from 'react';
+import React from 'react';
+import ResourceTypeContext from '../filters/resourceTypeContext';
+import KeyDropdown from '../labels/KeyDropdown';
+
+const DeploymentLabelConfigurationInput: FunctionComponent<Stage.Types.CustomConfigurationComponentProps<string[]>> = ({
+    value,
+    onChange,
+    name,
+    widgetlessToolbox
+}) => {
+    return (
+        <ResourceTypeContext.Provider value="deployments">
+            <KeyDropdown
+                onChange={(newValue: string[]) => {
+                    onChange(undefined, { name, value: newValue });
```

Comment:
When working on backporting your changes to 6.3 found out that there's one more bug.

If you set `labelsToShow` configuration parameter and then set it back to empty list, the widget won't render.
I didn't have time to dig into details, so ended up with quick/ugly fix:

```suggestion
                    onChange(undefined, { name, value: newValue || [] });
```

Please investigate and address this issue in the next PR.

Reply:
this means that typing of onChange for keyDropdown is bad

Reply:
created a ticket for that:

https://cloudifysource.atlassian.net/browse/RD-6345